distributed/scheduler.py

```diff
@@ -2059,6 +2059,93 @@ def transition_no_worker_processing(self, key, stimulus_id):
                 pdb.set_trace()
             raise
 
+    def transition_no_worker_erred(
```

Comment:
This is a mash-up of the best parts of `transition_no_worker_processing` and `transition_processing_erred`. We could refactor this to reduce code duplication and facilitate future changes, but I do not want to change too much here since the fate of this transition is TBD.

Reply:
XREF: #7346